public/docs/_examples/pipes/dart/lib/exponential_strength_pipe.dart

```diff
@@ -3,24 +3,18 @@ import 'dart:math' as math;
 import 'package:angular2/angular2.dart';
 
 /*
-* Raise the value exponentially
-* Takes a value that defaults to 0 and an exponent argument that defaults to 1.
-* Checks for value to be a string or number.
-* Usage:
-*   value | exponentialStrength:exponent
-* Example:
-*   {{ 2 |  exponentialStrength:10}}
-*   formats to: 1024
-*/
-
+ * Raise the value exponentially
+ * Takes an exponent argument that defaults to 1.
+ * Usage:
+ *   value | exponentialStrength:exponent
+ * Example:
+ *   {{ 2 |  exponentialStrength:10}}
+ *   formats to: 1024
+ */
 @Pipe(name: 'exponentialStrength')
-@Injectable()
+@Injectable() // FIXME(chalin): unnecessary?
```

Comment:
> The HeroService doesn't have any dependencies at the moment. Add the decorator anyway. It is a "best practice" to apply the @Injectable() decorator ​from the start​ both for consistency and for future-proofing.

From https://angular.io/docs/ts/latest/tutorial/toh-pt4.html

Event though it's not necessary, per style guide it should be added to all classes that are supposed to be injected.